submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,174 @@
+
+Moving on from producing steam to turning it into power, for every heat exchanger produces 103.09 steam per second while every steam turbine consumes 60, hence the ratio for them is 103.09/60 = 1.718 turbines for every exchanger. If there is a single steam network across the entire plant, our example plant would need 104 * 1.718 = 178.672, or 179 turbines. However, the example actually has 12 steam networks with either 7 or 12 heat exchangers, which respectively would need 7z1.718 = 12.026 (so 12, because the 13th turbine would have almost no utilization) or 12 * 1.718 = 20.616 (so 21) turbines. 
+
+A popular simple alternative for steam turbine placement is attaching 2 to each heat exchanger without connecting anything with pipes. This provides an excess steam consumption capacity that remains unused unless your plant has steam storage (discussed below).
+
+And so we have all the component counts covered. If you want to learn more about the mathematics involved here, or if you want a calculator to do most of it for you, the  [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) is the resource I'd recommend.
+
+### What do we consider when placing the components?
+
+In terms of where to build the nuclear plant as a whole, it is quite beneficial to build near or on water, because the plant needs tons of it. Being close to uranium is optional because necesary amounts can be easil transported via trains.
+
+When placing down the reactors, we already know about harnessing the neighbor bonus and  calculating it using SRE diagrams. Given that we want to connect as many neighbors as possible, it would make sense to just have a huge spaceless grid of reactors, but the disadvantage of that is there being no space to set up a system for automatically refueling any of the reactors in the middle, unless you [a mod for that](https://mods.factorio.com/mod/logistics-reactor-next). Hence the most popular reactor arrangement is 2xN, where all reactors except for the corner ones have 3 neighbors, hence each has at least 1 open side for inserter access. However, you can get as creative as you want and leave 2xN behind once you have a steady supply of U-235 such that the neighbor bonus is not so critical anymore.

```

Comment:
However, you can get as creative as you want and leave 2xN behind once you have a steady supply of U-235 such that the neighbor bonus is not so critical anymore.
-> well, losing a complete reactor equivalent by going away from the 2xN design will result if you requirering more reactors overall? then the production being excedentary seems to not be a point for me here (but I can be wrong :) )

Reply:
Well yes it becomes performance optimization vs aesthetic preferences and some people may strictly prefer one over the other. I'll emphasize that there may be trade-offs.